src/modules/0.0.22/aws_s3/index.ts

```diff
@@ -9,6 +9,7 @@ import { policiesAreSame } from '../../../services/aws-diff';
 import { AWS, crudBuilder2, crudBuilderFormat } from '../../../services/aws_macros';
 import { Context, Crud2, MapperBase, ModuleBase } from '../../interfaces';
 import { Bucket } from './entity';
+import { S3CleanBucketRpc } from './rpcs';
 
 class BucketMapper extends MapperBase<Bucket> {
```

Comment:
Do we want to move the Mapper into a `/mapper` directory with this change since the `/rpcs` directory has also been made?

Or perhaps move the singular `rpc` function in here, too?